api/common.ts

```diff
@@ -1,5 +1,14 @@
 export const baseURL = 'http://localhost/api'
 
 export const headers = {
-    'Content-Type': 'application/json',
-  }
\ No newline at end of file
+  'Content-Type': 'application/json',
+}
+
+export const getHeaders = () => {
```

Comment:
so we're gonna use the headers object for unauthenticated endpoints and getHeaders for auth ones, right? in that case let's rename getHeaders to getAuthedHeaders or something along those lines so it's clear when to use each one